karma.start.js

```diff
@@ -4,6 +4,8 @@ var store, DSUtils, dsHttpAdapter, dsLocalStorageAdapter, p1, p2, p3, p4, p5, p6
 var Post, User, Organization, Comment, Profile;
 var user1, organization2, comment3, profile4;
 var comment11, comment12, comment13, organization14, profile15, user10, user16, user17, user18, organization15, user19, user20, comment19, user22, profile21;
+var Item, Client;
```

Comment:
Just declare your Item and Client resources and your test data right in the test itself, rather than modifying `karma.start.js`